src/components/application_manager/test/resumption/resumption_data_test.cc

```diff
@@ -499,6 +500,7 @@ void ResumptionDataTest::SetCommands() {
 
     for (uint32_t j = 0; j < kCountOfChoice_; ++j) {
       char vr[12];
+      // cppcheck-suppress invalidPrintfArgType_sint
```

Comment:
@Ypostolov Do not suppress this issue. You can find solution at several comment above.